ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ChatReply.qml

```diff
@@ -47,7 +47,21 @@ Loader {
                 visible: repliedMessageType != Constants.imageType
                 anchors.top: lblReplyAuthor.bottom
                 anchors.topMargin: 5
-                text: Emoji.parse(Utils.linkifyAndXSS(repliedMessageContent), "26x26");
+                text: `<html>`+
+                    `<head>`+
```

Comment:
I found out in Eric's PR that you don't actually need the `html`, `body` and `head` tags https://github.com/status-im/nim-status-client/pull/1329/files#diff-d3fc1d4710c58e34d358243c5f8cce911c16bb950a184bed301dc1211cfab01cR70